src/site/content/en/blog/optimizing-content-efficiency-eliminate-downloads/index.md

```diff
@@ -1,20 +1,22 @@
 ---
-title: Eliminating Unnecessary Downloads
-description: >
+title: Eliminate unnecessary downloads
+description: |
   You should audit your resources periodically to ensure that each resource is helping deliver a better user experience.
 authors:
   - ilyagrigorik
+  - jlwagner
 date: 2014-03-31
-updated: 2018-08-17
+updated: 2023-07-06
 ---
 
-### Summary {: .hide-from-toc }
+## Summary
 
 - Inventory your own assets and third-party assets on your pages.
 - Measure the performance of each asset: its value and its technical performance.
 - Determine if the resources are providing sufficient value.
+- Understand the effect of unnecessary downloads on [Core Web Vitals](/vitals/) and supporting metrics.
```

Comment:
I wonder if this summary would be better moved to the end and this first section should be titles "Introduction" (but without the 4 bullet points which would be in Summary at the end)?

Reply:
Yeah, good callout! I'll rearrange some stuff and add another commit after going through some of this other feedback.